### PR TITLE
setting the dont_run to true based on the meta data added to the xclb…

### PR DESCRIPTION
**Pull request by venkatp-xilinx**
setting the dont_run to true based on the meta data added to the xclbin. This is pretty much used by the vck190 pcie platform

### CODE REVIEW

**Comment by dayeh-xilinx** [comment]
retest this please. Failed on boardfarm host cold reboot time out.

File: src/runtime_src/core/pcie/emulation/hw_em/generic_pcie_hal2/shim.cxx [hunk 1]
```diff
@@ -726,7 +727,25 @@ namespace xclhwemhal2 {
...
+        }
+      }
+      else {
+        mSimDontRun = simDontRun;
+      }
+    }
+    else {
+      mSimDontRun = simDontRun;
+    }
+
```
Comment (chvamshi-xilinx) [inline src/runtime_src/core/pcie/emulation/hw_em/generic_pcie_hal2/shim.cxx:747]: As discussed, Please simplify this code.

Comment (venkatp-xilinx) [reply src/runtime_src/core/pcie/emulation/hw_em/generic_pcie_hal2/shim.cxx:740]: committed the change as per the review comments

**Comment by dayeh-xilinx** [comment]
retest this please

**Pull request merged**